composer.json

```diff
@@ -30,16 +27,16 @@
     }
   },
   "require": {
-    "php": ">=5.6",
+    "php": ">=7.1",
     "smarty/smarty": "^3.1",
-    "nikic/php-parser": "^3",
+    "nikic/php-parser": "^4",
     "twig/twig": "^1.34 || ^2.0",
-    "symfony/twig-bridge": "^3.0"
+    "symfony/twig-bridge": "^3.4|^4.0"
   },
   "require-dev": {
-    "symfony/framework-bundle": "^3.0",
-    "symfony/translation": "^3.0",
-    "doctrine/common":  "~2.5.3",
-    "phpunit/phpunit": "^5"
+    "doctrine/common": "~2.10.0",
+    "phpunit/phpunit": "^5.7",
+    "symfony/framework-bundle": "^3.4|^4.0",
```

Comment:
Isn't it too soon to add Symfony 4 support? I'm worried that we may be introducing regressions as composer will pick the higher version for tests while it will use the lower one in PS.

Reply:
it's never too soon as 3.4 and 4.0 apis are the same, but ok let's revert that

Reply:
Done